### PR TITLE
update landcover documentation

### PR DESCRIPTION
**Pull request by marooned20**
Pull Request
============

> Tracked in JIRA by [UP-4585](https://up42.atlassian.net/browse/UP-4585)

### Scope of the PR

This PR contains recent changes made into LandCover block to be able to handle 4 or 5 classes for classefication.

### Checklist

- [x] Checked spelling and grammar
- [x] Provided code samples where relevant
- [x] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

File: source/up42-blocks/processing/land-cover.rst [hunk 1]
```diff
@@ -19,14 +19,18 @@ Land cover classification or segmentation is the process of assigning each of
...
   1. Water
   2. High vegetation (including trees)
   3. Low vegetation (including bushes and grass)
-  4. Urban (including roads and buildings)
+  4. Barren Land
+  5. Urban (including roads and buildings)
 
-Based on an independent test dataset we reached an Accuracy of 0.75 and a Jaccard of 0.22.
+Based on an independent test dataset we reached an Accuracy of 0.75 and a Jaccard of 0.22 when trained with 4 classes.
+When trained with 5 classes using the same test dataset, we achieved an Accuracy
```
Comment (rodrigoalmeida94) [inline source/up42-blocks/processing/land-cover.rst:33]: Missing some numbers here?

**Review by rodrigoalmeida94 — APPROVED** [review]
Nice!

File: source/up42-blocks/processing/land-cover.rst (hunk 1 above)
Comment (marooned20) [reply source/up42-blocks/processing/land-cover.rst]: lol, fixed it

**Review by teoEnache — APPROVED** [review]
lgtm

**Pull request merged**